kubectl-pg/README.md

```diff
@@ -0,0 +1,53 @@
+# kubectl-pg
+
+### Installtion of kubectl pg plugin
+
+Clone the kubectl pg plugin and build from the source using ```go install``` this will generate kubectl pg plugin's executable in ```go/bin/kubectl-pg.exe```
+
+### To list all commands avaialble in kubectl pg
+
+```kubectl pg --help``` (or) ```kubectl pg```
+
+### To validate postgresql CRD creation
```

Comment:
> To validate postgresql CRD exists

this basically means the operator pod managed to start, so our operator is installed.